offchain/graphql-server/Cargo.toml

```
@@ -15,6 +15,7 @@ path = "src/schema/generate_schema.rs"
 [dependencies]
 http-health-check = { path = "../http-health-check" }
 rollups-data = { path = "../data" }
+log = { path = "../log" }
```

Just keeping them in alphabetic order:
```suggestion
log = { path = "../log" }
rollups-data = { path = "../data" }
```

---

Done at 9e1b6f8e7bc477c334f4e109cf3bb7a1642bc44d